android/signin/.gitignore

```diff
@@ -7,3 +7,4 @@ build/
 *.apk
 *.aar
 *.zip
+google-services.json
```

Comment:
Could you add this to the other repos that use the google-services.json, Like gcm?